website/src/css/custom.css

```diff
@@ -150,3 +157,17 @@ div[class^='announcementBar_'] {
 .red > a {
   color: red;
 }
+
+.sr-only {
```

Comment:
what does "sr-only" mean?

Reply:
screen-readers. Hide an element visually only, and still accessible to assistive technology like screen readers

Reply:
I see thanks
Maybe add a comment or a more explicit className because there's no chance I'll remember this in a year 😅